src/main/java/com/whalespottingjava/controllers/AdminController.java

```diff
@@ -26,4 +32,23 @@ public String getAdminPage(Model model) {
         model.addAttribute("sightings", sightingService.getAllPendingSightings());
         return "admin";
     }
+
+    @PostMapping("/admin")
+    @ResponseStatus(HttpStatus.CREATED)
+    public String postSightings(@RequestBody List<AdminApprovalRequest> list, Model model) {
```

Comment:
I think the function name postSightings can easily get confused with someone for example bulk posting whale sightings. This is the endpoint admins call to approve/reject sightings, so perhaps postAdminApprovalRequest?